static/js/main.js

```diff
@@ -241,17 +242,23 @@ function addScreenshot(screenshot) {
                             value='${time_input_value}' />
                 </div>
                 <div>
-                    <button onclick='flipImage("${screenshot.id}")'>Flip Image</button>
+                    <span>
+                        <button onclick='flipScreenshot("${screenshot.id}", "x")'>Vertical Flip</button>
+                        <button onclick='flipScreenshot("${screenshot.id}", "y")'>Horizontal Flip</button>
+                    </span>
                 </div>
+                <div>
+                    <button onclick='deleteScreenshot("${screenshot.id}")'>Delete</button>
+                </div> 
             </div>
         </div>
     `;
 
     document.querySelector('.screenshot-area').insertAdjacentHTML('beforeend', html);
 }
 
-// Flip an image in the screenshot area by adding it the the canvas flipped horizontally.
-function flipImage(screenshot_id) {
+// Flip a screenshot by adding it the the canvas flipped horizontally or vertically.
+function flipScreenshot(screenshot_id, axis) {
```

Comment:
Let's change axis to a boolean like `flip_vertical`.
Since there are only two possible options for flipping, I think a boolean would be more clean (and eliminate error cases).